webrtc.html

```diff
@@ -2765,6 +2745,40 @@
           </dd>
         </dl>
 
+        <dl class="idl" title="dictionary RTCRtpCapabilities">
+          <dt>sequence&lt;RTCRtpCodecCapability&gt; codecs</dt>
+
+          <dd>
+            <p>
+              Supported codecs.
+            </p>
+          </dd>
+
+          <dt>sequence&lt;RTCRtpHeaderExtensionCapability&gt; headerExtensions</dt>
+
+          <dd>
+            <p>
+              Supported RTP header extensions.
+            </p>
+          </dd>
+        </dl>
+
+        <dl class="idl" title="dictionary RTCRtpCodecCapability">
+          <dt>DOMString name</dt>
+          <dd>
+            <p>The codec MIME type. Valid types are listed in [IANA-RTP-2].
```

Comment:
List discussion seemed to converge on a more complex representation here.... please review.